src/yang/ietf-bgp.yang

```diff
@@ -924,11 +928,156 @@ module ietf-bgp {
           }
         }
       }
+
       container peer-groups {
         description
           "Configuration for BGP peer-groups";
-        uses bgp-peer-group-list;
```

Comment:
I understand your intent here.  I'd like some time to stare at the contents to see if we can try to address the issue I opened to know if we can still leverage a common grouping to avoid duplication of code.

Reply:
That is entirely possible, and I can see myself defining some of the items as groupings. But let us get the sets to be common before I create them.

Reply:
Created a common grouping to avoid duplication of code.